### PR TITLE
Automated cherry pick of #126167: Hot fix for panic on schema conversion.

### PR DESCRIPTION
**Pull request by cici37**
Cherry pick of #126167 on release-1.29.

#126167: Hot fix for panic on schema conversion.

For details on the cherry pick process, see the [cherry pick requests](https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md) page.

```release-note

```

### CODE REVIEW

**Comment by liggitt** [comment]
/lgtm
/approve
cc @kubernetes/release-managers

File: staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/google/cel-go/cel"
 	"github.com/google/cel-go/checker"
 	"github.com/google/cel-go/common/types"
+	"github.com/pkg/errors"
```
Comment (liggitt) [inline staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go:27]: oh... don't use pkg/errors, use stdlib `errors`, and drop the go.mod change

Comment (cici37) [reply staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go]: Updated and here is the changes into current release: https://github.com/kubernetes/kubernetes/pull/126308

**Comment by saschagrunert** [comment]
@liggitt https://github.com/kubernetes/kubernetes/pull/126300 and https://github.com/kubernetes/kubernetes/pull/126302 are not LGTM'ed

**Comment by liggitt** [comment]
oops, forgot to sweep them all. tagged.

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**